File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/http/controllers/trainer/model/request/CreateReportRequest.kt
package pt.isel.leic.ptgest.http.controllers.trainer.model.request

import jakarta.validation.constraints.NotEmpty
import java.util.UUID

data class CreateReportRequest(
    val traineeId: UUID,
    @field:NotEmpty(message = "Report cannot be empty")
    val report: String,
    val visibility: Boolean
)

File: code/js/src/views/user/TrainerViews/models/reports/CreateReportRequest.ts
class CreateReportRequest{
    traineeId: string
    report: string
    visibility: boolean

    constructor(traineeId: string, report: string, visibility: boolean){
        this.traineeId = traineeId
        this.report = report
        this.visibility = visibility
    }
}
export default CreateReportRequest
